ckanext/push_errors/logging.py

```diff
@@ -67,21 +63,31 @@ def push_message(message, extra_context={}):
     default_title = 'PUSH_ERROR *{site_url}* \nv{push_errors_version} - CKAN {ckan_version}\n{now} user: {user}\n'
     title = toolkit.config.get('ckanext.push_errors.title', default_title)
 
-    message = title.format(**ctx) + "\n" + message
-    ctx['message'] = message
+    formated_message = title.format(**ctx) + "\n" + message
+    ctx['message'] = formated_message
+
+    # Si no hay URL configurada, registra el mensaje en los logs
+    if not url:
+        log.warning('push-errors: No URL configured, logging message locally.')
+        log.error(f'Local error log: {formated_message}')
+        return
```

Comment:
Do not leave if no URL
```suggestion

    if not url:
        log.warning('push-errors: No URL configured, logging message locally.')
```